### DISCUSSION TITLE
4/5 프로젝트 시작

### DISCUSSION THREAD

**Discussion by 42kko** [42webserv/42webserv#3] [General]
- 월, 수, 목 13시에 모이는걸 원칙으로합니다.
- 다음주 월요일(4월10일)부터 시작하며 개인약속은 최대한 정리합니다.
- c++에서 컨벤션사용이 가능하다면 사용해서 코드스타일은 통일합니다.

4월 10일까지 pdf를 읽고 해석까지 해서 어떤식으로 과제를 진행할지 각자 생각을해오는걸로

**Comment by get6** [comment]
나는 settings.json에서 아래와 같이 설정해서 사용 중!

```c++
"[cpp]": {
	"editor.defaultFormatter": "ms-vscode.cpptools",
	"editor.formatOnSave": true,
},
```
